plugin/src/main/java/net/dzikoysk/funnyguilds/feature/command/InvitePlayersCompleter.java

```diff
@@ -0,0 +1,55 @@
+package net.dzikoysk.funnyguilds.feature.command;
+
+import com.google.common.collect.Lists;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import net.dzikoysk.funnycommands.commands.CommandUtils;
+import net.dzikoysk.funnycommands.resources.Completer;
+import net.dzikoysk.funnycommands.resources.Context;
+import net.dzikoysk.funnyguilds.config.PluginConfiguration;
+import net.dzikoysk.funnyguilds.user.User;
+import net.dzikoysk.funnyguilds.user.UserManager;
+import org.bukkit.Bukkit;
+import panda.std.stream.PandaStream;
+
+public class InvitePlayersCompleter implements Completer {
+
+    private final PluginConfiguration configuration;
+    private final UserManager userManager;
+
+    public InvitePlayersCompleter(PluginConfiguration configuration, UserManager userManager) {
+        this.configuration = configuration;
+        this.userManager = userManager;
+    }
+
+
+    @Override
+    public String getName() {
+        return "invite-players";
+    }
+
+    @Override
+    public List<String> apply(Context context, String prefix, Integer limit) {
+        String[] args = context.getArguments();
+
+        if (args.length != 1) {
+            return Lists.newArrayList("");
+        }
+
+        return CommandUtils.collectCompletions(
+                Stream.concat(
```

Comment:
PandaStream ma w sobie metodę concat, trzeba to sprawdzić jeszcze, ale jeśli dobrze myślę - mógłbyś spokojnie zrobić PandaStream z tymi graczami, dokleic do niego pojedynczy String i wtedy toList - powinno wyjść krócej